src/__tests__/routes/__test__/protectedRoute.test.js

```diff
@@ -0,0 +1,24 @@
+import decode from "jwt-decode";
+import Authenticate from "../../../routes/protectedRoutes";
+
+describe("protected routes", () => {
+  it("should render without crashing", () => {
+    expect(Authenticate("token")).toEqual(false);
+  });
+
+  it("should render without crashing", () => {
```

Comment:
Please change this message to something different because we are having two tests with the same name